manager/src/test/java/io/gingersnapproject/mysql/MySQLResources.java

```diff
@@ -1,43 +1,49 @@
 package io.gingersnapproject.mysql;
 
 import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+import org.testcontainers.containers.BindMode;
 import org.testcontainers.containers.MySQLContainer;
 import org.testcontainers.utility.MountableFile;
 
+import java.nio.file.Path;
 import java.util.Map;
 
 public class MySQLResources implements QuarkusTestResourceLifecycleManager {
 
-    private static final String IMAGE = "mysql:8.0.31";
-    private MySQLContainer<?> db;
-
-    @Override
-    public Map<String, String> start() {
-
-
-        db = new MySQLContainer<>(IMAGE)
-                .withUsername("gingersnap_user")
-                .withPassword("password")
-                .withExposedPorts(MySQLContainer.MYSQL_PORT)
-                .withCopyFileToContainer(MountableFile.forClasspathResource("setup.sql"), "/docker-entrypoint-initdb.d/setup.sql");
-        db.start();
-
-        return Map.of(
-                "quarkus.datasource.db-kind", "MYSQL",
-                "quarkus.datasource.username", db.getUsername(),
-                "quarkus.datasource.password", db.getPassword(),
-                "quarkus.datasource.reactive.url", String.format("mysql://%s:%d/debezium", db.getHost(), db.getMappedPort(MySQLContainer.MYSQL_PORT)),
-                "gingersnap.rule.us-east.key-type", "PLAIN",
-                "gingersnap.rule.us-east.plain-separator", ":",
-                "gingersnap.rule.us-east.select-statement", "select fullname, email from customer where id = ?",
-                "gingersnap.rule.us-east.connector.schema", "debezium",
-                "gingersnap.rule.us-east.connector.table", "customer"
-        );
-    }
-
-    @Override
-    public void stop() {
-        if (db != null)
-            db.stop();
-    }
+   public static final String RULE = "us-east";
+   private static final String IMAGE = "mysql:8.0.31";
+
+   private static final String HOST_TMP = Path.of(System.getProperty("java.io.tmpdir"), "mysql").toString();
+   private static final String CONTAINER_DATA_DIR = "/var/lib/mysql";
+
+   private MySQLContainer<?> db;
+
+   @Override
+   public Map<String, String> start() {
+      db = new MySQLContainer<>(IMAGE)
+            .withUsername("gingersnap_user")
+            .withPassword("password")
+            .withExposedPorts(MySQLContainer.MYSQL_PORT)
+            .withFileSystemBind(HOST_TMP, CONTAINER_DATA_DIR, BindMode.READ_WRITE)
```

Comment:
be aware that I'm binding the `/tmp/` directory into the container. It makes the test way faster (I have the docker data on an HDD) and the data is there if you re-run the test.
I can remove it or use a system property to set it up.

Reply:
No objections from me, that's great to know it speeds things up.

Reply:
Great idea IMO too. Maybe we should apply it to all tests.

Reply:
Sounds like a good plan to me. The faster the testsuite the better